### PR TITLE
ocp4 - AU-2 - update to complete and update narrative

### PR DESCRIPTION
**Pull request by itewk**
peer reviewed by:
* kevindoingbizas

### CODE REVIEW

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 1]
```diff
@@ -28,29 +28,31 @@
...
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: We'll need to be a bit more specific about what OpenShift can audit out of the box. The most common questions revolve around the ability to meet requirements from :

- OSPP Configuration Annex
https://www.niap-ccevs.org/MMO/PP/-442ConfigAnnex-/

- ICD 500-27 (scroll to pdf pg7)
https://www.dni.gov/files/documents/FOIA/DF-2016-00213.pdf

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 2]
```diff
@@ -28,29 +28,31 @@
...
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: We'll need to be a bit more specific about what OpenShift can audit out of the box. The most common questions revolve around the ability to meet requirements from :

- OSPP Configuration Annex
https://www.niap-ccevs.org/MMO/PP/-442ConfigAnnex-/

- ICD 500-27 (scroll to pdf pg7)
https://www.dni.gov/files/documents/FOIA/DF-2016-00213.pdf

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 3]
```diff
@@ -28,29 +28,31 @@
...
     - key: c
       text: |
+        Coordinating the security audit function with other
+        organizational entities is outside the scope of
+        system configuration.
+
         For rationale regarding why the auditable events are deemed to be
         adequate to support after-the-fact investigations of security incidents,
         please contact your countries Common Criteria representative:
         https://www.commoncriteriaportal.org/ccra/members/
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: Coordination is part of AU-2(b), but not AU-2(c). Reference NIST language:

``b. Coordinates the security audit function with other organizational entities requiring audit-related information to enhance mutual support and to help guide the selection of auditable events;

c. Provides a rationale for why the auditable events are deemed to be adequate to support after-the-fact investigations of security incidents; and
``

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 4]
```diff
@@ -28,29 +28,31 @@
...
     - key: d
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: Response to this control is a gray area. We're often asked to document what events OpenShift will audit in a default installation, which would be part of a control response here (e.g. "every time a user does a deployment, pushes code, or deletes a pod, an audit event is triggered in the default installation of OpenShift. In addition to the default audit events, the NIST National Checklist for OpenShift 4.x adds the following.....")

Ref NIST language:
``d. Determines that the following events are to be audited within the information system: [Assignment: organization-defined audited events (the subset of the auditable events defined in AU-2 a.) along with the frequency of (or situation requiring) auditing for each identified event].``

Comment (shawndwells) [reply openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: Documenting what default audit events are enabled would be a great way to make incremental progress on this control.

**Comment by shawndwells** [comment]
Bump to @kevindoingbizas and @itewk

**Comment by itewk** [comment]
Heyo team, can you take this over working with @shawndwells please.

@curt-matthews
@mradecker
@kevindoingbizas
@thamill
@mtclemente
@kpeeples
@dwinchell
@mammerman
@mlcassatt
@siliconjesus
@mertel-rh
@kevin-anderson-redhat

**Comment by shawndwells** [comment]
@itewk Due to no response to feedback for approximately 25 days, this PR is being closed. Feel free to re-open if you find time to work on this!

**Comment by itewk** [comment]
@mammerman can you take ownership of this?

**Pull request closed without merging**